core/lib/Thelia/Core/PropelInitService.php

```diff
@@ -389,8 +389,6 @@ public function init($force = false)
                 $theliaDatabaseConnection->useDebug(true);
             }
         } catch (\Throwable $th) {
```

Comment:
The catch clause is not needed. The finally clause alone is enough.

Reply:
maybe the cache should be reset at this time to avoid hidden errors at next boot